seacrowd/sea_datasets/khmer_alt_pos/khmer_alt_pos.py

```diff
@@ -0,0 +1,165 @@
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses
+
+_CITATION = """\
+@article{10.1145/3464378,
+author = {Kaing, Hour and Ding, Chenchen and Utiyama, Masao and Sumita, Eiichiro and Sam, Sethserey and Seng, Sopheap and Sudoh, Katsuhito and Nakamura, Satoshi},
+title = {Towards Tokenization and Part-of-Speech Tagging for Khmer: Data and Discussion},
+year = {2021},
+issue_date = {November 2021},
+publisher = {Association for Computing Machinery},
+address = {New York, NY, USA},
+volume = {20},
+number = {6},
+issn = {2375-4699},
+url = {https://doi.org/10.1145/3464378},
+doi = {10.1145/3464378},
+abstract = {As a highly analytic language, Khmer has considerable ambiguities in tokenization and part-of-speech (POS) tagging processing. This topic is investigated in this study. Specifically, a 20,000-sentence Khmer corpus with manual tokenization and POS-tagging annotation is released after a series of work over the last 4 years. This is the largest morphologically annotated Khmer dataset as of 2020, when this article was prepared. Based on the annotated data, experiments were conducted to establish a comprehensive benchmark on the automatic processing of tokenization and POS-tagging for Khmer. Specifically, a support vector machine, a conditional random field (CRF), a long short-term memory (LSTM)-based recurrent neural network, and an integrated LSTM-CRF model have been investigated and discussed. As a primary conclusion, processing at morpheme-level is satisfactory for the provided data. However, it is intrinsically difficult to identify further grammatical constituents of compounds or phrases because of the complex analytic features of the language. Syntactic annotation and automatic parsing for Khmer will be scheduled in the near future.},
+journal = {ACM Trans. Asian Low-Resour. Lang. Inf. Process.},
+month = {sep},
+articleno = {104},
+numpages = {16},
+keywords = {annotated data, tokenization, POS-tagging, Khmer, machine learning}
+}
+"""
+
+_LANGUAGES = ["khm"]  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+_LOCAL = False
+
+_DATASETNAME = "khmer_alt_pos"
+
+_DESCRIPTION = """\
+The data contains 20,000-sentence Khmer corpus with manual tokenization and POS-tagging annotation.
+"""
+
+_HOMEPAGE = "https://www2.nict.go.jp/astrec-att/member/mutiyama/ALT/km-nova-181101/README.txt"
+
+_LICENSE = Licenses.CC_BY_4_0.value
```

Comment:
from readme file in issue homepage,
I find the license is `Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0) License`
```suggestion
_LICENSE = Licenses.CC_BY_NC_SA_4_0.value
```